client/go/outline/electron/main.go

```diff
@@ -115,11 +122,11 @@ func main() {
 	if len(*args.transportConfig) == 0 {
 		printErrorAndExit(platerrors.PlatformError{Code: platerrors.InvalidConfig, Message: "transport config missing"}, exitCodeFailure)
 	}
-	clientResult := outline.NewClient(*args.transportConfig)
-	if clientResult.Error != nil {
-		printErrorAndExit(clientResult.Error, exitCodeFailure)
+
+	client, err := newOutlineClient(*args.transportConfig, *args.adapterIP, *args.adapterIndex)
```

Comment:
How do you pass the fwmark for Linux?

Is this no longer needed for Linux?

Reply:
We don't use any code here in Linux any more, we use library calls: `establishVPN`.